### DISCUSSION TITLE
Is there a plug-n-play frontend for llama-cpp-python[server]

### DISCUSSION THREAD

**Discussion by siddhsql** [abetlen/llama-cpp-python#341] [Q&A]
From the README I understand 

> llama-cpp-python offers a web server which aims to act as a drop-in replacement for the OpenAI API.

Is there some frontend that I can layer on top of the server to get a Web UI?

**Comment by jmtatsch** [comment]
https://github.com/mckaywrigley/chatbot-ui

**Comment by siddhsql** [comment]
Thanks. Ya, saw it yesterday.
